### ISSUE TITLE
about the training data

### ISSUE THREAD

**Issue by lyq216398** [wenyyu/Image-Adaptive-YOLO#79]
Did you split the training set into the training set and the validation set to select the best epoch or did you just use all the training sets to select the last epoch for testing?
